webapp/lib/platform.dart

```diff
@@ -59,6 +59,19 @@ bool isUserSignedIn() {
 
 typedef CollectionListener(List<DocSnapshot> changes);
 
+Future<List<String>> get activeProjects async =>
+    (await _firestoreInstance.collection('projects').where('archived', '==', false).get()).docs.map((doc) => doc.id).toList();
+
+Future<Map<String, List<String>>> get getProjectsDrivers async {
```

Comment:
consistency with `activeProjects`, as well as not needing to be prefixed by `get` if it's a getter method
```suggestion
Future<Map<String, List<String>>> get projectsDrivers async {
```

Reply:
Yeah. Renamed it. I think initially I wasn't using a getter so I forgot to rename it